src/agent/samples/policy/yaml/deployment/deployment-front.yaml

```diff
@@ -2,56 +2,46 @@
     spec:
       nodeSelector:
         kubernetes.io/os: linux
       runtimeClassName: kata-cc
       containers:
-        - name: azure-vote-back
-          image: "mcr.microsoft.com/oss/bitnami/redis:6.0.8"
-          env:
-            - name: ALLOW_EMPTY_PASSWORD
-              value: "yes"
-          resources:
-            requests:
-              cpu: 100m
-              memory: 128Mi
-            limits:
-              cpu: 250m
-              memory: 256Mi
+        - name: frontend
+          image: "mcr.microsoft.com/azurelinux/base/nginx:1"
           ports:
-            - containerPort: 6379
-              name: redis
```

Comment:
Are we testing the port name using a some other YAML files?

Reply:
Good catch, I missed that. But yes https://github.com/microsoft/kata-containers/blob/msft-main/src/agent/samples/policy/yaml/kubernetes/conformance/csi-hostpath-plugin.yaml#L238